java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -142,14 +152,48 @@ class DatabaseImpl(
             )
         }
 
+    @VisibleForTesting
+    fun getCollection(storageKey: StorageKey, schema: Schema): DatabaseData.Collection =
+        readableDatabase.useTransaction {
+            val db = this
+            val collectionId = requireNotNull(getCollectionId(storageKey, db)) {
+                "Collection at storage key $storageKey does not exist."
+            }
+
+            val values = mutableSetOf<Reference>()
+            rawQuery(
+                """
+                    SELECT entity_refs.entity_id, entity_refs.backing_storage_key
+                    FROM collection_entries
+                    JOIN entity_refs ON collection_entries.value_id = entity_refs.id
+                    WHERE collection_entries.collection_id = ?
+                """.trimIndent(),
+                arrayOf(collectionId.toString())
+            ).forEach {
+                values.add(Reference(
```

Comment:
Nit: place `Reference(` on a new line, and insert a new line between the `))` on line 177.

Reply:
Done